plugins/playstation/src/playstation-plugin.vala

```diff
@@ -0,0 +1,32 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class Games.PlayStation : Object, Plugin {
+	private const string FINGERPRINT_PREFIX = "playstation";
+	private const string MIME_TYPE = "application/x-cue";
```

Comment:
https://wiki.libretro.com/index.php?title=Beetle/Mednafen_PSX#cite_note-4 mentions that Mednafen supports CUE/BIN (as above), CUE/TOC and "CCD/IMG/SUB".

This mime-type is also problematic, because it's just a "disc image" format, not something specific to the PSX. There should really be a more precise mime-type with magic (and added to shared-mime-info).